scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -412,27 +412,27 @@ def _ignore_old_dead_container( container, created_before=None ):
 
     return False
 
-def _get_containers(client, ignore_container=None, ignore_pod=None, restrict_to_container=None, logger=None,
+def _get_containers(client, ignore_container=None, ignored_pod=None, restrict_to_container=None, logger=None,
                     only_running_containers=True, running_or_created_after=None, glob_list=None, include_log_path=False, k8s_cache=None,
                     k8s_include_by_default=True, k8s_namespaces_to_exclude=None, ignore_pod_sandboxes=True, current_time=None):
     """Queries the Docker API and returns a dict of running containers that maps container id to container name, and other info
-        @param client: A docker.Client object
-        @param ignore_container: String, a single container id to exclude from the results (useful for ignoring the scalyr_agent container)
-        @param ignore_container: QualifiedPod, a pod name and namespace to exclude from results (useful for ignoring the scalyr_agent container)
-        @param restrict_to_container: String, a single continer id that will be the only returned result
-        @param logger: scalyr_logging.Logger.  Allows the caller to write logging output to a specific logger.  If None the default agent.log
+        @param: client: A docker.Client object
```

Comment:
Nitpick - the rest of the code base seems to follow [epytext format](http://epydoc.sourceforge.net/manual-fields.html)  so no '`:`' between `@param` and the parameter name.

`@param client:` rather than `@param: client:`